internal/locate/region_request.go

```diff
@@ -1277,9 +1277,9 @@ func (s *replicaSelector) onServerIsBusy(
 		// Mark the server is busy (the next incoming READs could be redirect
 		// to expected followers. )
 		ctx.Store.markAlreadySlow()
-		if s.canFallback2Follower() {
-			return true, nil
-		}
+	}
+	if s.canFallback2Follower() {
```

Comment:
As @MyonKeminta has mentiond, there are situations the `backoff` is still needed, like:
- All the replicas in this region have reported busy or have other information indicating that the node is slow
- If the node that will be retried next is likely to be slow when retrying

It's better to clearly specify the conditions under which backoff can be skipped and then skip it, like
```
if conditions
   skip backoff
```
rather than deciding which situations require backoff, 
```
if conditions
  do backoff
```
to reduce the risk of changes.

Reply:
Nice catch, done, PTAL